app/src/ui/discard-changes/discard-changes-dialog.tsx

```diff
@@ -88,6 +81,27 @@ export class DiscardChanges extends React.Component<
     )
   }
 
+  private renderCheckbox() {
```

Comment:
Could we rename this to something a little bit less generic? Such as `renderConfirmDiscardChanges`? And similarly rename the `onCheckboxChanged` to `onConfirmDiscardChangesChanged`?